src/main/java/com/minecolonies/coremod/entity/ai/citizen/netherworker/EntityAIWorkNether.java

```diff
@@ -0,0 +1,958 @@
+        itemsNeeded.add(GuardGearBuilder.buildGearForLevel(ARMOR_LEVEL_CHAIN, ARMOR_LEVEL_DIAMOND, LEATHER_BUILDING_LEVEL_RANGE, DIA_BUILDING_LEVEL_RANGE));
+        itemsNeeded.add(GuardGearBuilder.buildGearForLevel(ARMOR_LEVEL_LEATHER, ARMOR_LEVEL_IRON, LEATHER_BUILDING_LEVEL_RANGE, IRON_BUILDING_LEVEL_RANGE));
+        itemsNeeded.add(GuardGearBuilder.buildGearForLevel(ARMOR_LEVEL_LEATHER, ARMOR_LEVEL_CHAIN, LEATHER_BUILDING_LEVEL_RANGE, CHAIN_BUILDING_LEVEL_RANGE));
+        itemsNeeded.add(GuardGearBuilder.buildGearForLevel(ARMOR_LEVEL_LEATHER, ARMOR_LEVEL_GOLD, LEATHER_BUILDING_LEVEL_RANGE, GOLD_BUILDING_LEVEL_RANGE));
+
+    }
+
+    @Override
+    public Class<BuildingNetherWorker> getExpectedBuildingClass()
+    {
+        return BuildingNetherWorker.class;
+    }
+  
+    @Override
+    public IAIState getStateAfterPickUp()
+    {
+        return START_WORKING;
+    }
+
+    @Override
+    public boolean canBeInterrupted()
+    {
+        return !worker.isInvisible();
+    }
+
+    @Override
+    protected IAIState decide()
+    {
+        if(job.isInNether())
+        {
+            if (!worker.isInvisible())
+            {
+                worker.setInvisible(true);
+            }
+            return NETHER_AWAY;
+        }
+        if (worker.isInvisible())
+        {
+            worker.setInvisible(false);
+        }
+
+        IAIState crafterState = super.decide();
+
+        if(crafterState != IDLE && crafterState != START_WORKING)
+        {
+            return crafterState;
+        }
+
+        // Get Armor if available. 
+        // This is async, so we could go to the nether without it. 
+        checkAndRequestArmor();
+
+        // Check for materials needed to go to the Nether: 
+        IRecipeStorage rs = getOwnBuilding().getFirstModuleOccurance(BuildingNetherWorker.CraftingModule.class).getFirstRecipe(ItemStack::isEmpty);
+        if(rs != null)
+        {
+            for(ItemStorage item : rs.getInput())
+            {
+                checkAndRequestMaterials(item.getItemStack(), item.getAmount());
+            }
+        }
+
+        // Make sure we have a stash of some food 
+        checkAndRequestFood(16);
+
+        // Get other adventuring supplies. These are required. 
+        // Done this way to get all the requests in parallel
+        boolean haveAxe = checkForToolOrWeapon(ToolType.AXE);
+        boolean havePick = checkForToolOrWeapon(ToolType.PICKAXE);
+        boolean haveShovel = checkForToolOrWeapon(ToolType.SHOVEL);
+        boolean haveSword = checkForToolOrWeapon(ToolType.SWORD);
+        boolean haveLighter = checkForToolOrWeapon(ToolType.FLINT_N_STEEL);
+        if(haveAxe || havePick || haveShovel || haveSword || haveLighter)
+        {
+            worker.getCitizenData().setIdleAtJob(true);
+            return IDLE;
+        }
+
+        if (currentRecipeStorage == null)
```

Comment:
when can this happen?

Reply:
This is the start of a trip. When we have all of the basic equipment, and nothing else is blocking us is when we hit this point.  Further along, if we don't have the actual trip recipe inputs, we'll check to see if the portal is supposed to be closed, and isn't.